ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -87,28 +85,14 @@ fun LandingScreen(
 
                 Chip(
                     onClick = onClickWatchList,
-                    label = {
-                        Text(
-                            stringResource(R.string.list_of_watches_button_label),
-                            maxLines = 1,
-                            overflow = TextOverflow.Ellipsis
-                        )
-                    },
-                    modifier = Modifier.fillMaxWidth()
+                    label = stringResource(R.string.list_of_watches_button_label)
                 )
             }
             for (listItem in menuItems) {
                 item {
                     Chip(
                         onClick = listItem.clickHander,
-                        label = {
-                            Text(
-                                listItem.name,
-                                maxLines = 1,
-                                overflow = TextOverflow.Ellipsis
-                            )
-                        },
-                        modifier = Modifier.fillMaxWidth()
```

Comment:
Isn't fill required?

Reply:
It is [set](https://github.com/google/horologist/blob/564353a0232038764768a9d0e1874ea5065d8793/compose-material/src/main/java/com/google/android/horologist/compose/material/Chip.kt#L219) in Chip, since the beginning, but thinking in making it more flexible, maybe adding a flag to choose it to not be set. Thoughts?

Reply:
I'm not sure I've ever used without full width. So that's ok. It's probably more surprising it breaks the typical layout responsibilities.

Let's leave for now?